### PR TITLE
send recsys powered-by

### PR DESCRIPTION
**Pull request by jessopb**

### CODE REVIEW

**Comment by jessopb** [comment]
Had to change urisByQuery to resultsByQuery so that I could store the recsys id with the query since it changes by load balancer.

File: ui/redux/selectors/search.js [hunk 1]
```diff
@@ -42,9 +43,9 @@ export const makeSelectSearchUris = (query: string): ((state: State) => Array<st
     if (query) {
       query = query.replace(/^lbry:\/\//i, '').replace(/\//, ' ');
       const normalizedQuery = createNormalizedSearchKey(query);
-      return byQuery[normalizedQuery];
+      return byQuery[normalizedQuery] && byQuery[normalizedQuery]['uris'];
     }
-    return byQuery[query];
+    return byQuery[query] && byQuery[query]['uris'];
   });
```
Comment (infinite-persistence) [inline ui/redux/selectors/search.js:49]: Wunderbar broke a bit (not sure if it's specifically due to this line or not)

Steps to reproduce:
1. Homepage
2. Search something in Wunderbar and press enter (I used "manchester").  We'll end up in the Search Page.
3. Clear Wunderbar and type "manchester" again but don't press enter.  
    - In the drop down, `Most Supported` appears, but `Search results` is blank.  Reverted the change and `Search results` will appear.

File: ui/redux/selectors/search.js [hunk 2]
```diff
@@ -28,7 +29,7 @@ export const selectIsSearching: (state: State) => boolean = createSelector(selec
 
 export const selectSearchUrisByQuery: (state: State) => { [string]: Array<string> } = createSelector(
```
Comment (infinite-persistence) [inline ui/redux/selectors/search.js:30]: Maybe a rename would be good to reflect the actual returned object.

**Review by infinite-persistence — COMMENTED** [review]
:+1: Good to go with the following issues left:

**Review by infinite-persistence — APPROVED** [review]

**Pull request merged**